src/main/java/com/amazon/opendistroforelasticsearch/ad/model/ADTask.java

```diff
@@ -87,6 +93,10 @@
 
     private String coordinatingNode = null;
     private String workerNode = null;
+    private DetectionDateRange detectionDateRange = null;
+    private List<Entity> entity = null;
```

Comment:
should this be called something like `entities` or `entityList`?

Reply:
Good question, this is to keep consistent with current code, check this one https://github.com/opensearch-project/anomaly-detection/blob/main/src/main/java/com/amazon/opendistroforelasticsearch/ad/model/AnomalyResult.java#L91

We can rename this for all places with a dedicated PR to make the change clean and easy to review.